README.md

```diff
@@ -50,6 +50,7 @@ Store is used to map an upload's fingerprint with the corresponding upload URL.
 | Name | Backend | Dependencies |
 |:----:|:-------:|:------------:|
 | MemoryStore | In-Memory | None |
+| LeveldbStore | LevelDB | None |
```

Comment:
How would you like me to document the dependencies? By supplying the import paths?